src/index.js

```diff
@@ -1 +1,46 @@
-export default "TODO";
+import * as babylon from "babylon";
+
+// Parse options values.
+const _parseOpts = (opts) => Object.keys(opts)
+  // Parse the matched replacement expression and
+  // https://github.com/babel/babylon/issues/210
+  .map((key) =>
+    [key, babylon.parse(`expr=${opts[key]}`).program.body.pop().expression.right])
```

Comment:
I would add, however, that a suboptimal solution (from a UX perspective) would be to accept the actual AST nodes as options rather than the JS expression itself.